src/backend/marsha/core/models/video.py

```diff
@@ -388,3 +388,71 @@ class Meta:
         verbose_name = _("signs language track")
         verbose_name_plural = _("signs language tracks")
         indexes = [NonDeletedUniqueIndex(["video", "language"])]
+
+
+class Thumbnail(BaseModel):
```

Comment:
Maybe call this model Image to make it more generic and allow using it for other purposes?

A thumbnail is when we use an image on the video.